cloudconfig/cloudinit/cloudinit_win.go

```diff
@@ -120,3 +122,8 @@ func (cfg *windowsCloudConfig) addRequiredPackages() {
 // updateProxySettings is defined on the AdvancedPackagingConfig interface.
 func (cfg *windowsCloudConfig) updateProxySettings(proxy.Settings) {
 }
+
+// AddNetworkConfig is defined on the NetworkingConfig interface.
+func (cfg *windowsCloudConfig) AddNetworkConfig(interfaces []network.InterfaceInfo) error {
+	return nil
```

Comment:
Does this and centos return nice errors in these cases? Let's not leave the user hanging.

Reply:
That's just code refactoring - I don't want to change the logic in any way that's not needed for this PR.